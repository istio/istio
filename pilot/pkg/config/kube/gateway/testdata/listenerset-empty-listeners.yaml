apiVersion: gateway.networking.k8s.io/v1beta1
kind: GatewayClass
metadata:
  name: istio
spec:
  controllerName: istio.io/gateway-controller
---
apiVersion: gateway.networking.k8s.io/v1beta1
kind: Gateway
metadata:
  name: parent-gateway
  namespace: istio-system
spec:
  allowedListeners:
    namespaces:
      from: All
  addresses:
    - value: istio-ingressgateway
      type: Hostname
  gatewayClassName: istio
  listeners: []
---
apiVersion: gateway.networking.x-k8s.io/v1alpha1
kind: XListenerSet
metadata:
  name: single-entry-http
  namespace: istio-system
spec:
  parentRef:
    name: parent-gateway
    kind: Gateway
    group: gateway.networking.k8s.io
  listeners:
    - name: first
      hostname: first.foo.com
      protocol: HTTP
      port: 80
