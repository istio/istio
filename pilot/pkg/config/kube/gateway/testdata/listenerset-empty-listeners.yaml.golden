apiVersion: networking.istio.io/v1
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-semantics: gateway
    internal.istio.io/gateway-service: istio-ingressgateway.istio-system.svc.domain.suffix
    internal.istio.io/parent-namespace: istio-system
    internal.istio.io/parents: XListenerSet/single-entry-http/first.istio-system
  creationTimestamp: null
  name: single-entry-http-istio-autogenerated-k8s-gateway-first
  namespace: istio-system
spec:
  servers:
  - hosts:
    - istio-system/first.foo.com
    port:
      name: default
      number: 80
      protocol: HTTP
---
